### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+ - If the PR is a major change, merge is postponed until the next release cycle.
+   - Dev calls out on the issue that it is a major change and it will be merged after branching.
+   - Once the current release is branched the PR can be merged into `master`.
+
+#### PR Merged
+
+After a PR is merged:
+- External Contributions: PM closes the [Help Wanted] issue and related JIRA ticket.
+- Internal Contributions: Core committer resolves the JIRA ticket.
+- PM follows up for docs and changelog, and QA for release tests.
```
Comment (mkraft) [inline CONTRIBUTING.md]: @jasonblais What are the absolute minimum things a contributor would need to know _before_ submitting a pull request?

Does specific knowledge of the labels system and how they're used by the PMs qualify as required knowledge that would preclude a contributor from submitting a PR?

What about other steps of the review process? Do they _need_ to know all steps? Some steps? Any steps at all?

Comment (jasonblais) [reply CONTRIBUTING.md]: I can think about it some more, but I just copied this file from https://github.com/mattermost/mattermost-server/blob/master/CONTRIBUTING.md

We do have plans to make the contribution guidelines brief, an early draft can be found here http://mattermost-developer-documentation.s3-website-us-east-1.amazonaws.com/commits/ca4b55e329e9e4c22fb71b41b45f748053795b76/contribute/getting-started/contribution-checklist/

Comment (jasonblais) [reply CONTRIBUTING.md]: The review process could be separated into its own page and linked from CONTRIBUTING.md if a contributor is interested to learn more about it.

Comment (mkraft) [reply CONTRIBUTING.md]: Ah so this is just a copy/paste of the existing docs with a new process in the works: sounds like some good improvements are in the works and that's great! 👍

**Comment by cpanato** [comment]
need rebase

**Comment by jasonblais** [comment]
@cpanato I don't know how to rebase :/

**Comment by cpanato** [comment]
done

**Comment by jasonblais** [comment]
Thank you :)

**Pull request merged**